projects/plugins/super-cache/tests/e2e/package.json

```diff
@@ -23,17 +23,17 @@
 		"test:run": "jest --runInBand"
 	},
 	"devDependencies": {
-		"@jest/globals": "29.1.1",
+		"@jest/globals": "^28.1.0",
 		"@typescript-eslint/eslint-plugin": "5.36.1",
 		"@typescript-eslint/parser": "5.36.1",
 		"axios": "0.27.2",
 		"cheerio": "1.0.0-rc.12",
 		"dotenv": "16.0.2",
 		"eslint": "8.22.0",
 		"eslint-plugin-jest": "26.8.7",
```

Comment:
Same here. All the eslint stuff seems to come in from the monorepo level.

Reply:
Righto. 1bcfa4e1ff